app/src/main/java/com/kodeco/android/coordplot/MapBox.kt

```diff
@@ -0,0 +1,43 @@
+package com.kodeco.android.coordplot
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun MapBox(xPercent: Float, yPercent: Float, modifier: Modifier = Modifier) {
+    val configuration = LocalConfiguration.current
+
+    var boxSize = 300
+    when (configuration.orientation) {
+        Configuration.ORIENTATION_LANDSCAPE -> {
+            boxSize = 280
+        }
+    }
```

Comment:
Great job here configuring a different size based on the orientation!

Fun fact, you can consolidate this a bit and use a `val` instead of `var` here too:

```suggestion
    val boxSize = when (configuration.orientation) {
        Configuration.ORIENTATION_LANDSCAPE -> {
            boxSize = 280
        }
        else -> 300
    }
```